python/py-scipy/Portfile

```diff
@@ -7,10 +7,10 @@ PortGroup               github 1.0
 PortGroup               compilers 1.0
 PortGroup               compiler_blacklist_versions 1.0
 
-github.setup            scipy scipy 1.5.0 v
-checksums               rmd160  bae805468c0781ea8fff87f25b7c3384dcd0875e \
-                        sha256  b31b616b4571f19a4c1a925b1b19f4eb325d098265692d88ea386537aa3742ab \
-                        size    19831004
+github.setup            scipy scipy 1.5.1 v
+checksums               rmd160  112cc52964a9c7890328ea53045bfac024309f7f \
```

Comment:
Let's move this to a separate commit from Fortran tweaks.

Reply:
Done.